cmd/nfs-subdir-external-provisioner/provisioner.go

```diff
@@ -47,13 +47,36 @@ type nfsProvisioner struct {
 	path   string
 }
 
+type pvcMetadata struct {
+	data        map[string]string
+	labels      map[string]string
+	annotations map[string]string
+}
+
+func (meta *pvcMetadata) stringParser(str string) string {
+	pattern := regexp.MustCompile(`{pvc\.((labels|annotations)\.(.*?)|.*?)}`)
```

Comment:
if the pattern is static, better to move it to global scope

Reply:
Got it